src/Components/ToDoList/ToDoList.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+// import { ToDoItem } from '../ToDoItem/ToDoItem';
+
+type Props = {
+  todos: Todo[]
+};
+
+export const TodoList: React.FC<Props> = React.memo(
+  ({ todos }) => {
+    return (
```

Comment:
```suggestion
  ({ todos }) => (
```
not critical, just you to know, you may do it in this way